src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -177,6 +179,29 @@ private String getPartitionKeyFallback(String partitionKeyFallback) {
 		}
 	}
 
+	private void writeEncryptedMapToJSON(String jsonMapName, LinkedHashMap<String, Object> data, boolean includeNullField, String encryption_key, String secret_key) throws IOException{
+		JsonGenerator generator = jsonGeneratorThreadLocal.get();
+		JSONObject obj = new JSONObject();
```

Comment:
let's avoid this and creating a new dependency for maxwell.  Instead, make `writeMapToJson` accept a `JsonGenerator` object, and have it write a raw map into the generator.  (you'll have to remove the part where `writeMapToJson` writes the `data` key).  The regular code path will write directly into the main generator object, but the encryption code path will create a new generator for the map and capture the bytes back and encrypt them. 

make sense?